terraform-infra-approvals/div-case-orchestration-service.json

```diff
@@ -0,0 +1,8 @@
+{
+  "resources": [
+  ],
+  "module_calls": [
+
+    {"source":  "git@github.com:hmcts/cnp-module-webapp?ref=revert-159-tf-0.12.x"}
```

Comment:
why do we need this?

Reply:
testing something on div demo - doesn't apply the app service on clean build

Reply:
state file issue?

Reply:
was originally, but removed remote state file and all infra - from clean infra/state it didn't attempt to build the app service when `enable_ase = true`